app/Console/Kernel.php

```diff
@@ -76,6 +79,20 @@ protected function schedule(Schedule $schedule): void
                         ->onOneServer()
                         ->withoutOverlapping(15);
 
+                    $schedule->call(function () use ($tenant) {
+                        $tenant->execute(function () {
+                            Bus::batch([new ServiceMonitoringJob(ServiceMonitoringFrequency::OneHour)])->dispatch();
+                        });
+                    })
+                        ->hourly();
+
+                    $schedule->call(function () use ($tenant) {
+                        $tenant->execute(function () {
+                            Bus::batch([new ServiceMonitoringJob(ServiceMonitoringFrequency::TwentyFourHours)])->dispatch();
```

Comment:
I don't think we need the batch functionality at the moment for these. So let's just do a normal `dispatch`